docs/docs/HowTo/environments.md

```diff
@@ -0,0 +1,107 @@
+# Manage environments
+
+Environments in Klaw are similar to application environments that act as
+a wrapper over clusters. Each environment in Klaw is associated with a
+cluster, and there are three types of environments available:
+
+-   Kafka
+-   Schema Registry
+-   Kafka Connect
+
+## Prerequisite
+
+To manage environments in Klaw, including adding, deleting, or updating,
+you need to log in using a SUPERADMIN account or an account with the
+`ADD_EDIT_DELETE_ENVS` permission.
+
+## Add new environment
+
+To add a new environment in Klaw, follow these steps:
+
+1.  Log in to Klaw.
+2.  Navigate to the **Environments** tab and select **Environments**.
+3.  Select **Add Environment**.
+4.  Select Add Environment for the type of environment you want to add.
+    
+
+To add a **Kafka environment**, enter the following details in the **Add Kafka Environment** screen:
+
+        -   Environment name : Choose an environment from the
+            available dropdown list.
+        -   Select cluster: Choose the Kafka cluster you want to
```

Comment:
@harshini-rangaswamy should we add a space in between each of these bullet points for readability?

Reply:
The list displayed here is rendered as a code block rather than a sub-list. The spacing should be automatically adjusted to the appropriate list style spacing by aligning or modifying the indentation.

Reply:
@aindriu-aiven  reformatted bullets and added headings